Update repl.it running code in readme (#240)
Androz2091/scratch-for-discord: README.md

## Run On Repl.it

You can run your bot on repl.it. To do so, click the download link at the top right corner of the website. Import these files in your repl.it project. Run the following code in the "SHELL" tab:

> - `npm install discord.js`
> - `npm install easydb`

Reviewer: ```suggestion
```

## Run On Repl.it

> You can run your bot on repl.it. To do so, click the download link at the top right corner of the website. Import these files in your repl.it project. Run the following code in the "SHELL" tab:

Reviewer: ```suggestion
You can run your bot on repl.it. To do so, click the download link at the top right corner of the website. Import these files in your repl.it project and run the following commands in the "SHELL" tab:
```

> You can run your bot on repl.it. To do so, click the download link at the top right corner of the website. Import these files in your repl.it project and run the following commands in the "SHELL" tab:

Reviewer: ```suggestion
You can run your bot on repl.it. To do so, click the download link at the top right corner of the website. Import these files in your repl.it project, and run the following commands in the "SHELL" tab:
```
